mongodb-aggregate-query-support-spring/src/test/java/com/github/krr/mongodb/aggregate/support/tests/AggregationAnnotationTests.java

```diff
@@ -75,6 +74,35 @@ public void mustCreateMatchAggregationOperation() {
     assertNotNull(result);
   }
 
+  @Test
+  public void mustDeserializeBothISOAndMillisForDate() throws ParseException {
```

Comment:
Make these into parameterized test.  Rather than having multiple scenarios in one test.